user-managerd.pro

```diff
@@ -45,12 +45,13 @@ DISTFILES += \
     org.sailfishos.usermanager.service \
     org.sailfishos.usermanager.xml \
     rpm/user-managerd.spec \
+    tmp-guest_home.mount \
     userdel_local.sh
 
 target.path = /usr/bin/
 
-systemd.files = dbus-$${DBUS_SERVICE_NAME}.service
-systemd.path = /usr/lib/systemd/system/
+systemd.files = dbus-$${DBUS_SERVICE_NAME}.service tmp-guest_home.mount
+systemd.path = /lib/systemd/system/
```

Comment:
$$[QT_INSTALL_LIBS]/systemd/system/

New files should be going all to the /usr/lib/systemd/system